### PR TITLE
mans_koans

### PR DESCRIPTION
**Pull request by mikhailman**

### CODE REVIEW

**Comment by mikhailman** [comment]
@anatoliliotych чекните меня)

File: triangle.rb [hunk 1]
```diff
@@ -15,6 +15,21 @@
 #
 def triangle(a, b, c)
   # WRITE THIS CODE
+  if ([a, b, c].select {|i| i <= 0}.size > 0)
```
Comment (anatoliliotych) [inline triangle.rb]: make it oneliner

File: about_scoring_project.rb [hunk 2]
```diff
@@ -35,43 +35,43 @@ def score(dice)
 
 class AboutScoringProject < Neo::Koan
   def test_score_of_an_empty_list_is_zero
-    assert_equal 0, score([])
+    assert_equal nil, score([])
```
Comment (anatoliliotych) [inline about_scoring_project.rb]: implement score method!!!!

File: about_dice_project.rb [hunk 3]
```diff
@@ -2,9 +2,17 @@
...
 # Implement a DiceSet Class here:
 #
-# class DiceSet
-#   code ...
-# end
+ class DiceSet
+   def initialize
+     @values = []
+   end
+   def values
```
Comment (anatoliliotych) [inline about_dice_project.rb]: use attr_reader

File: about_dice_project.rb [hunk 4]
```diff
@@ -2,9 +2,17 @@
...
+     @values
+   end
+   def roll(number)
+      @values = Array.new (number) {rand 1..6}
```
Comment (anatoliliotych) [inline about_dice_project.rb]: check style guide

**Comment by anatoliliotych** [comment]
@mikhailman respond to code review!

**Pull request closed without merging**